src/settings.qml

```diff
@@ -183,7 +183,8 @@ import Qt.labs.settings 1.0
 
             property double horizon_gr7_cadence_multiplier: 1.0
 
-            property int  fitshow_user_id: 0x006E13AA
+            property int  fitshow_user_id: 0x13AA
```

Comment:
@p3g4asus is the id change something important?

Reply:
Not so much. In the last version the two most significant bytes are ignored because user id should be 2 bytes long. The other bytes are for different purposes (and they were for different purposes also for non anyrun treadmills).